hbase-common/src/main/resources/hbase-default.xml

```diff
@@ -2062,4 +2062,9 @@ possible configurations would overwhelm and obscure the important.
       hitting the namenode, if the DFSInputStream's block list is incomplete.
     </description>
   </property>
+  <property>
+    <name>hbase.fs.reader.warn.time</name>
+    <value>100</value>
```

Comment:
Is 100ms a good default value? Maybe we should set it to 0 or -1, means disable the logging, and if users want, they can set it to the value they want?

Reply:
thanks for reviewing, i set the default value to -1 in the lastest commit in this MR